static/elements/chromedash-guide-metadata.js

```diff
@@ -95,6 +93,31 @@ export class ChromedashGuideMetadata extends LitElement {
     this.fetchData();
   }
 
+  /* Add the form's event listener after Shoelace event listeners are attached
+   * see more at https://github.com/GoogleChrome/chromium-dashboard/issues/2014 */
+  updated() {
```

Comment:
Does updated() get called multiple times?
If each call adds a listener, do we need to remove the old listeners?

Reply:
Yes updated will be called multiple times, so we need to either remove old listeners or not add another if one is already active.

Reply:
I did think about and test this.

The only thing that would trigger updated() after first update currently in this component is the `editing` attribute. However, every time when `editing` changes from false to true, a new form is attached to the DOM because form rendering is toggled by this variable as well:
```js
${this.editing ? this.renderEditForm() : this.renderReadOnlyTable()}
```

I also tested putting console.log() in the handleFormSubmission(), and found that there's always only one active listener (I assume the old listenter is garbage collected with the old form when `editing` changes from true to false).